docusaurus/docs/engineering/knowledge-base/development-lifecycle.md

```diff
@@ -4,154 +4,239 @@ sidebar_position: 7
 
 # Development Lifecycle
 
-To look at/download BobaBoard's code, check out our guide to [Setting Up your Coding Environment](/docs/engineering/start-developing/before-you-start).
-You might also be interested in [Git Notes](/docs/engineering/legacy-pages/git-notes)
+> //TODO: Consider making this doc a folder with multiple sections and moving to the Volunteering section
```

Comment:
You can do 
```
:::TODO
Text
:::
```

Similar to note. In theory it should have a custom display. It might be broken, and we should make a note to fix it.

Reply:
Looks broken indeed, they just look like the INFO callouts. I filed an issue in this repo for it.